website/content/docs/k8s/compatibility.mdx

```diff
@@ -15,9 +15,9 @@ Consul Kubernetes versions all of its components (`consul-k8s` CLI, `consul-k8s-
 
 | Consul version | Compatible `consul-k8s` versions | Compatible Kubernetes versions | Compatible OpenShift versions |
 | -------------- | -------------------------------- | -------------------------------| ----------------------------- |
+| 1.17.x         | 1.3.x                            | 1.25.x - 1.28.x                | 4.11.x - 4.14.x               |
```

Comment:
OpenShift remains unchanged since 4.14 is almost out and 4.14 is out later in the year.